packages/section/src/section.tsx

```diff
@@ -8,10 +8,10 @@ export const Section = React.forwardRef<HTMLTableElement, SectionProps>(
       <table
         align="center"
         width="100%"
-        {...props}
         border={0}
         cellPadding="0"
         cellSpacing="0"
+        {...props}
         ref={ref}
         role="presentation"
```

Comment:
Can you also move the `role="presentation"` one?

Reply:
Done